lib/Maintenance/Tasks/CleanupFieldcollectionTablesTask.php

```diff
@@ -26,11 +26,22 @@
  */
 class CleanupFieldcollectionTablesTask implements TaskInterface
 {
+    const PIMCORE_FIELDCOLLECTION_CLASS_DIRECTORY = PIMCORE_CLASS_DIRECTORY . '/DataObject/Fieldcollection/Data';
+
     private LoggerInterface $logger;
+    private $mapLowerToCamelCase = [];
 
     public function __construct(LoggerInterface $logger)
     {
         $this->logger = $logger;
+        
+        $files = array_diff(scandir(self::PIMCORE_FIELDCOLLECTION_CLASS_DIRECTORY), array('..', '.'));
+        foreach ($files as $file) {
+            $classname = str_replace('.php','',$file);
+            $class = '\\Pimcore\\Model\\DataObject\\Fieldcollection\\Data\\' . $classname;
+            $object = new $class();
+            $this->mapLowerToCamelCase[strtolower($classname)] = $object->getType();
```

Comment:
```suggestion
            $classname = str_replace('.php','',$file);
            $this->mapLowerToCamelCase[strtolower($classname)] = $classname;
```
Can the class name be different than the type?